Mod Menu Helper/assets/modmenu/lang/tr_tr.json

```diff
@@ -3,42 +3,49 @@
   "modmenu.dropInfo.line2": "diğerleri desteklenmemektedir.",
   "modmenu.summaryTranslation.advancementinfo": "Daha büyük ve akıllı başarım ekranı\n§9✎§8 Renkler, bilgi kutucuk büyüklüğü, ...",
   "modmenu.summaryTranslation.advancements-enlarger": "§c*§r Daha büyük başarım ekranı\n§8✎ (seçenekleri yok)",
-  "modmenu.summaryTranslation.animatica": "OptiFine'ın hareketli dokular\n§8✎ (sadece gelişmiş seçenekler)",
-  "modmenu.summaryTranslation.antighost": "Hayalet blokları /ghost\n§9✎§8 tuş atamasıyla düzelt.",
-  "isxander-main-menu-credits.antighost": "Hayalet blokları /ghost",
+  "modmenu.summaryTranslation.animatica": "OptiFine'ın hareketli dokuları\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.antighost": "Hayalet blokları /ghost ile düzelt\n§9✎§8 Tuş ataması",
+  "isxander-main-menu-credits.antighost": "Hayalet blokları /ghost ile düzelt",
   "modmenu.summaryTranslation.betterbeds": "§c*§r Hızlı yataklar\n§8✎ (seçenekleri yok)",
   "modmenu.summaryTranslation.bettermounthud": "Atın üstündeyken yemek ve XP'yi göster\n§8✎ (seçenekleri yok)",
   "modmenu.summaryTranslation.bobby": "§c*§r Sunuculardaki yüklü yığınları yüklü tutar\n§9✎§8 Varlıkları blokla, yığınları azalt, önbelleği koru, ...",
   "modmenu.summaryTranslation.borderlessmining": "Tam ekrandayken Alt+Tab desteği (opt-in)\n§9✎§8 Modu, macOS desteğini, özel boyut vb. aç/kapa",
   "modmenu.summaryTranslation.capes": "OptiFine veya diğerlerinden pelerinler\n§9✎§8 Türler, tercih edilen tür, elitra, ...",
   "modmenu.summaryTranslation.citresewn": "OptiFine'ın özel eşyaları\n§9✎§8 Modu aç/kapa, bozuk paketlere izin ver, ...",
   "modmenu.summaryTranslation.colormatic": "OptiFine'ın özel renkleri\n§9✎§8 Gökyüzü/yer boşluğunu temizle, ışık yumuşatma",
-  "modmenu.summaryTranslation.continuity": "OptiFine'ın bağlantılı dokuları\n§9✎§8 Modu, metodu, kaynak paketlerini aç/kapa",
-  "modmenu.summaryTranslation.cullleaves": "OptiFine'ın \"daha iyi ağaçları\"\n§9✎§8 Modu, kaynak paketini aç/kapa",
+  "modmenu.summaryTranslation.continuity": "OptiFine'ın bağlantılı dokuları\n§9✎§8 Modu, yöntemi, kaynak paketlerini aç/kapa",
+  "modmenu.summaryTranslation.cull-less-leaves": "Daha hızlı ağaç yaprakları\n§9✎§8 Modu aç/kapa, yaprak derinliği",
   "modmenu.summaryTranslation.cem": "OptiFine'ın özel varlıkları (alfa)\n§9✎§8 OptiFine desteği, model düzeltimi",
+  "modmenu.summaryTranslation.debugify": "Hataları düzzeltir, Mojang'ın teşhis göndermesini kapatır.\n§9✎§8 Daha fazla bilgi için FO wikisine bakın",
```

Comment:
"kapatır" does not fit to the first row :(

Reply:
How can I solve this problem? There is no alternative

Reply:
Replace "düzeltir" with "çözer".

Reply:
'çözer" and "düzeltir" are not interchangeable

Reply:
I used "tanı" instead of "teşhis"
Maybe that can help.